docs/business-apps/power-automate/get-started/sharepoint-actions-triggers.md

```diff
@@ -3,51 +3,51 @@ title: Microsoft SharePoint Connector for Power Automate
+Triggers the flow when you create a file in a document library, and returns only the custom file properties associated with that file.
 
 ### When a file is created in a folder
 
-Triggers the flow when a file is created in a SharePoint folder. This trigger does not run if a file is added or updated in a subfolder inside the folder this trigger is operating on. If the flow is required to trigger on subfolders, create different flows for one or more subfolder(s).
+Triggers the flow when you create a file in a SharePoint folder. This trigger does not run if you add or update a file in a subfolder inside the folder this trigger is operating on. If the flow is required to trigger on subfolders, create different flows for one or more subfolder(s).
 
 ### When a file is created or modified (properties only)
 
-Triggers the flow when a file is created, and each time the file properties are modified in a library. Returns only the custom file properties associated with that file.
+Triggers the flow when you create a file, and each time you modify the file properties in a library. Returns only the custom file properties associated with that file.
 
 ### When a file is created or modified in a folder (properties only)
 
-Triggers the flow when a file is created, and each time the file properties are modified in the selected SharePoint folder. The trigger does not activate if a file is added/updated in a subfolder. If the flow is required to trigger on subfolders, create different flows for one or more subfolder(s).
+Triggers the flow when you create a file, and each time you modify the file properties in the selected SharePoint folder. The trigger does not activate if you add or update a file in a subfolder. If the flow is required to trigger on subfolders, create different flows for one or more subfolder(s).
 
 ### When a file is deleted
 
-Triggers the flow when a file is deleted in a document library. You can optionally specify a folder to watch. When a folder is deleted, the trigger activates only once for the deleted folder, including its subfolders. To get the properties of the deleted file, this trigger must connect to the associated SharePoint site on which the trigger is epxected to run using a site collection user account.
+Triggers the flow when you delete a file in a document library. You can optionally specify a folder to watch. When you delete a folder, the trigger activates only once for the deleted folder, including its subfolders. To get the properties of the deleted file, you must connect this trigger to the associated SharePoint site on which the trigger is epxected to run, using a site collection user account.
 
 ### For a selected file
 
-Triggers the flow for a selected file in a document library.
+Triggers the flow for a file you select in a document library.
 
 ### When a site has requested to join a hub site
```

Comment:
Missing description. We can add the following:

_Triggers the flow when a site has requested to join a hub site. This triggers requires that approvals is configured and turned on for the hub site. Learn more [here on how to set up approvals in a hub site](https://support.microsoft.com/en-us/office/set-up-your-sharepoint-hub-site-e2daed64-658c-4462-aeaf-7d1a92eba098#bkmk_managesiteassociationapprovals)._